packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/add.html

```diff
@@ -0,0 +1,138 @@
+                </p>
+            </div>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_name' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding">
+
+            <oui-field  data-size="xl"
+                        data-help-text="{{ :: 'pci_projects_project_serving_namespace_models_add_name_help' | translate }}">
+                <input class="oui-input" type="text"
+                       id="name" name="id"
+                       data-ng-model="$ctrl.model.id"
+                       data-required
+                       data-ng-pattern="/^[\w-]+$/"/>
+            </oui-field>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_autoscaling' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding">
+
+            <span data-translate="pci_projects_project_serving_namespace_models_add_autoscaling_info"></span>
+
+            <oui-field data-label="{{:: 'pci_projects_project_serving_namespace_models_storage_scale_min' | translate }}"
+                       data-size="auto">
+                <oui-numeric min="1" max="1" model="$ctrl.core" name="test" disabled>
+                </oui-numeric>
+            </oui-field>
+            <oui-field data-label="{{:: 'pci_projects_project_serving_namespace_models_storage_scale_max' | translate }}"
+                       data-size="auto">
+                <oui-numeric min="3" max="3" model="$ctrl.max" name="test2" disabled>
+                </oui-numeric>
+            </oui-field>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{ :: 'pci_projects_project_serving_namespace_models_add_instances' | translate }}"
+            data-submit-text="{{ :: 'pci_projects_project_serving_namespace_models_common_add' | translate }}"
+            data-editable="!$ctrl.isAdding"
+            data-loading="$ctrl.isAdding">
+
+            <span data-translate="pci_projects_project_serving_namespace_models_add_instances_info"></span>
+
+            <div class="container-fluid px-0">
+                <div class="row">
+                    <div class="col-md-6 col-xl-4 my-3"
+                         data-ng-repeat="flavor in $ctrl.flavors track by flavor.id">
+                        <oui-select-picker
+                            data-name="{{:: flavor.id }}"
+                            data-model="$ctrl.model.flavor"
+                            data-label="{{:: flavor.description }}"
+                            data-variant="light"
+                            data-values="[flavor]"
+                            data-required>
+                            <oui-select-picker-section>
+                                <div class="text-right">
+                                    <span
```

Comment:
`span`+ `.d-block`=== `div` 🙂 

Also, you don't need the `span`, everything could be in the `div` above